dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +230,88 @@ def draw_chimera(G, **kwargs):
+    If the ``interaction_edges`` parameter is not None, then only display the couplers in that
+    list. If the ``embedded_graph`` parameter is not None, then only display the couplers between
+    chains with intended couplings according to the ``embedded_graph`` parameter.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a Chimera graph.
+        :term:`Chimera` :term:`graph` or a :term:`subgraph` of a Chimera graph.
 
     emb : dict
-        A dict of chains associated with each node in G.  Should be
-        of the form {node: chain, ...}.  Chains should be iterables
-        of qubit labels (qubits are nodes in G).
+        A dict of chains associated with each node in ``G`` and
+        of the form ``{node: chain, ...}``. Chains are iterables
+        of qubit labels (qubits are nodes in ``G``).
 
     embedded_graph : NetworkX graph (optional, default None)
-        A graph which contains all keys of emb as nodes.  If specified,
-        edges of G will be considered interactions if and only if they
-        exist between two chains of emb if their keys are connected by
-        an edge in embedded_graph
+        Graph which contains all keys of the ``emb`` parameter as nodes. If specified, then
+        the edges of ``G`` will be considered interactions if and only if they
+        exist between two chains of the ``emb`` parameter and if their keys are connected by
+        an edge in the ``embedded_graph`` parameter.
 
     interaction_edges : list (optional, default None)
-        A list of edges which will be used as interactions.
+        List of edges which will be used as interactions.
 
     show_labels: boolean (optional, default False)
-        If show_labels is True, then each chain in emb is labelled with its key.
+        If True, then each chain in the ``emb`` parameter is labelled with its key.
 
     chain_color : dict (optional, default None)
-        A dict of colors associated with each key in emb.  Should be
-        of the form {node: rgba_color, ...}.  Colors should be length-4
-        tuples of floats between 0 and 1 inclusive. If chain_color is None,
-        each chain will be assigned a different color.
+        A dict of colors associated with each key in the ``emb`` parameter and
+        of the form ``{node: rgba_color, ...}``. Colors must be length-4
+        tuples of floats between 0 and 1, inclusive. If None,
+        then each chain will be assigned a different color.
 
     unused_color : tuple (optional, default (0.9,0.9,0.9,1.0))
-        The color to use for nodes and edges of G which are not involved
-        in chains, and edges which are neither chain edges nor interactions.
-        If unused_color is None, these nodes and edges will not be shown at all.
+        Color to use for graph ``G``'s nodes that are not part of
+        chains and edges that are neither chain edges nor interactions.
```

Comment:
Ambiguous phrasing: it's unclear whether it is saying "part of chains and edges that are" or "... and edges that are"